Makefile

```diff
@@ -71,13 +73,14 @@ install: debianinstall genericinstall
 
 debianinstall:
 	sudo apt-get update -qq
-	sudo apt-get -y install libxml-libxml-perl libjson-perl libjson-xs-perl python-libvirt
+	sudo apt-get -y install libxml-libxml-perl libjson-perl libjson-xs-perl libwww-perl python-pip python3-pip python-setuptools python3-setuptools python-libvirt python3-libvirt
 
 suseinstall:
-	sudo zypper install perl-JSON-XS perl-libxml-perl python-pip libvirt-python
+	sudo zypper install perl-JSON-XS perl-libxml-perl python-pip python3-pip libvirt-python python3-libvirt-python
```

Comment:
if we add `libwww-perl` above, we should also add perl-CGI here

Reply:
Done: I assume you meant: `perl-libwww-perl`